agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AiConfigCustomizer.java

```diff
@@ -274,6 +274,10 @@ private static void enableInstrumentations(
       properties.put("otel.instrumentation.akka-http.enabled", "true");
       properties.put("otel.instrumentation.scala-fork-join.enabled", "true");
     }
+    if (config.preview.instrumentation.pekko.enabled) {
+      properties.put("otel.instrumentation.pekko-actor.enabled", "true");
+      properties.put("otel.instrumentation.pekko-http.enabled", "true");
```

Comment:
Pekko is a fork of Akka (because Akka is no longer open source) - it might be better to match Akka's setup - e.e by also including ` properties.put("otel.instrumentation.scala-fork-join.enabled", "true")`.

Reply:
Done. Thanks!